test/unit/authnz/test_custos_authnz.py

```diff
@@ -67,6 +67,7 @@ def setUp(self):
         self.test_nonce_hash = hashlib.sha256(self.test_nonce).hexdigest()
         self.test_code = "test-code"
         self.test_username = "test-username"
+        self.test_username_invalid = "test-username"
```

Comment:
Did you want to include a `@` here ?

Reply:
Argh, refactor fail; Good catch.  Yeah, I had test-username@foo.com but then I separated it into invalid/valid up here.